Interni/NormeDiProgetto/res/sections/processi_primari/processi_primari.tex

```diff
@@ -1,2 +1,243 @@
+\subsubsection{Aspettative}
+Il gruppo intende avviare e intrattenere un costante dialogo con il proponente per avere un feedback continuativo sul lavoro svolto ed in particolare:
+\begin{itemize}
+\item Determinare i bisogni che il committente si prefigge di soddisfare mediante il prodotto finale
+\item Stabilire vincoli sui requisiti richiesti
+\item Stimare le tempistiche di lavoro ed i costi 
+\item Effettuare una verifica continua del lavoro
+\item Prevenire in anticipo eventuali ambiguità o incertezze in merito al prodotto
+\item Concordare scelte relative alla qualifica del prodotto
+\end{itemize}
+
+\subsubsection{Studio di fattibilità}
+A seguito della presentazione dei capitolati d'appalto da parte dei proponenti il Responsbile di Progetto organizza riunioni tra i membri del gruppo al fine di condividere internamente opinioni sui capitolati stessi.
+Lo Studio di Fattibilità è redatto dagli analisti e per ogni capitolato deve indicare:
+\begin{itemize}
+\item \textbf{Informazioni Generali:} ovvero un elenco di informazioni di base che identifichino il progetto, il proponente e il committente;
+\item \textbf{Descrizione del capitolato:}  evidenziando le caratteristiche principali per il prodotto e gli obiettivi dello stesso;
+\item \textbf {Finalità del progetto:} le finalità richieste dal capitolato;
+\item \textbf {Tecnologie interessate:} le tecnologie da impiegare nello svolgimento del progetto;
+\item \textbf{Aspetti Positivi:} constatati in fase di analisi sulla base delle informazioni reperite in favore delle tecnologie richieste e del contesto di applicazione del prodotto finale;
+\item \textbf{Criticità e fattori di rischio:} esposizione di eventuali criticità o possibili tali rilevate nella fase di analisi contestualmente agli aspetti positivi del punto precedente
+\item \textbf{Conclusioni:} riassunto delle ragioni per cui il gruppo ha deciso di accettare o meno il capitolato in esame.
+\end{itemize}
+
+\subsubsection{Piano di Progetto}
+Deve essere redatto un Piano di Progetto da seguire durante lo svolgimento del progetto, in particolare deve contenere: 
+
+\begin{itemize}
+\item \textbf{Analisi dei Rischi:} vengono analizzati approfonditamente i rischi che potrebbero presentarsi e vengono preventivate delle modalità di mitigazione degli stessi. Viene inoltre stimata la probabilità con la quale questi possono presentarsi e il loro livello di gravità
+\item \textbf{Modello di Sviluppo:} descrizione del modello di sviluppo adottato 
+\item \textbf{Pianificazione:} descrizione della pianificazione preventiva dei tempi e delle attività del progetto.
+\item \textbf{Preventivo e consuntivo:}  viene fornita una stima del lavoro necessario per ciascuna fase ottenendo un preventivo per il costo totale,  fornendo un consuntivo di periodo relativo all'andamento rispetto a quanto preventivato.
+\end{itemize}
+
+\subsubsection{Piano di Qualifica}
```

Comment:
Credo che la struttura del piano di qualifica vada normata più approfonditamente in questa sezione

Reply:
In questa sezione, l'obiettivo è solamente descrivere la presenza di un documento. Il contenuto del piano di qualifica non è altro che l'insieme delle metriche utilizzate nelle norme di progetto. Quindi non c'è altro da scrivere. Forse si può rendere più leggibile?